docs/usage/docker.md

```diff
@@ -250,15 +250,19 @@ module.exports = {
 
 ##### Using long-lived service account credentials
 
-To access Google Container Registry (deprecated) or Google Artifact Registry you can use the JSON service account directly with `Basic` auth using `_json_key` as username and the service account as password.
+To access the Google Container Registry (deprecated) or the Google Artifact Registry, use the JSON service account with `Basic` authentication, and use the:
 
-Because JSON in JSON wrapping makes things more complex, avoid it completely by encoding the JSON service account beforehand.
+- `_json_key` as username
+- full Google Cloud Platform service account JSON as password
 
-Google Artifact Registry, but not Google Container Registry, supports `_json_key_base64` and a base64 encoded service account natively.
-If all your dependencies are on Google Artifact Registry, you can base64 encode and use the service account directly:
+To avoid JSON-in-JSON wrapping, which can cause problems, encode the JSON service account beforehand.
 
-1. Download your JSON service account and store it on your machine. Make sure that the service account has read (and only read) permissions to your artifacts.
-1. Base64 encode the service account credentials using `cat service-account.json | base64`
+Google Container Registry does not natively support `_json_key_base64` and a base64 encoded service account.
+Google Artifact Registry supports `_json_key_base64` and a base64 encoded service account natively.
```

Comment:
It's hard for me to keep the difference between GAR and GCR in my head while I'm reading the text.

Would it be better to split the document even more, and have two separate sections? So we'd have:

- one section about the Google Container Registry (GCR)
- one section about the Google Artifact Registry (GAR)

Also, the Google Container Registry is deprecated, could we just drop the GCR explanation completely?

Reply:
Sounds like we can keep the document as it is now then. 😄